csfieldguide/templates/interactives/scene-editor.html

```diff
@@ -48,7 +48,8 @@ <h5 id="scene-creation-title" class="col-4 offset-2 heading-line-text">{% trans
             </div>
           </div>
         </div>
-        <button id="apply-transformation" class="mt-2 btn btn-primary">{% trans "Apply" %}</button>
+        <button id="apply-transformation" class="mt-2 mx-2 btn btn-primary">{% trans "Apply" %}</button>
+        <button id="delete-button" class="mt-2 mx-2 d-none btn btn-outline-danger">{% trans "Delete" %}</button>
       </div>
 
```

Comment:
Lets add some hints about how we can pan :smile_cat: yay

Reply:
I have decided to not do this for three reasons:
- It's fairly self-explanatory; all three mouse buttons do something and most people would at least attempt it, you only didn't check RMB because you were used to it not doing anything.
- It's not all that important, most of the functionality you want is in orbiting the camera, which (LMB) is probably the second thing anyone would try
- Panning is not always possible on mobile, and adding instructions in those cases would be confusing. Plus I'm not sure how it would work on mac - do mac mice still only have the one button? (Shows how long it's been since I used one)

Reply:
Fair points :+1: